ledger/accountdb.go

```diff
@@ -101,7 +101,7 @@ var creatablesMigration = []string{
 // createNormalizedOnlineBalanceIndex handles accountbase/catchpointbalances tables
 func createNormalizedOnlineBalanceIndex(idxname string, tablename string) string {
 	return fmt.Sprintf(`CREATE INDEX IF NOT EXISTS %s
-		ON %s ( normalizedonlinebalance, address ) WHERE normalizedonlinebalance>0`, idxname, tablename)
+		ON %s ( normalizedonlinebalance, address, data ) WHERE normalizedonlinebalance>0`, idxname, tablename)
```

Comment:
@algorandskiy after talking to @brianolson maybe it's a good idea to just get rid of data, and not worry about inconsistent schemas. It would be better to make a ticket to follow up in another PR and get rid of this index altogether after the 320-rounds merge.